### DISCUSSION TITLE
Are you able to open up a website from the server to check how it looks?

### DISCUSSION THREAD

**Discussion by Basgamer999** [coder/code-server#5627] [Q&A]
Like you open up the .html file on local drive.
Are you able to have a discord bot boot too to check if it works instead of copying the files over again.

**Comment by jsjoeio** [comment]
You could do something like start a python server `python3 -m http.server` in your current directory then use `<domain>/proxy/<port>` to see the directory and then open the HTML file. Would that work?

**Reply by Basgamer999** [reply]
I am not sure if that works as i host it on petro panel. I need something as like remote file access on it so i can do IP:port/path to open up the file.

**Reply by jsjoeio** [reply]
hmm @code-asher any ideas?

**Reply by code-asher** [reply]
There is no good way to do this aside from the way @jsjoeio mentioned.

The ideal way might be to use the "live preview" extension but it does not yet work with code-server (the recent work on `asExternalUri` is a step forward to making it work though).
